nbstripout/_nbstripout.py

```diff
@@ -404,6 +404,9 @@ def main():
     parser.add_argument('files', nargs='*', help='Files to strip output from')
     args = parser.parse_args()
 
+    if not merge_configuration_file(args):
+        return 1
```

Comment:
This won't be needed if you make `merge_configuration_file` return `args` as per my other suggestions.